pkg/operator/ceph/csi/spec.go

```diff
@@ -192,6 +194,11 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 
 	tp.EnableCSIGRPCMetrics = fmt.Sprintf("%t", EnableCSIGRPCMetrics)
 
+	if os.Getenv("CSI_FORCE_CEPH_KERNEL_CLIENT") == "true" {
```

Comment:
What about running `strings.ToLower` on the env var? So that if someone uses `True` or `TRUE` it is covered.

Reply:
done